conda/recipes/libcuspatial/meta.yaml

```diff
@@ -56,13 +72,23 @@ outputs:
       run_exports:
         - {{ pin_subpackage("libcuspatial", max_pin="x.x") }}
       ignore_run_exports_from:
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% else %}
         - {{ compiler('cuda') }}
+        - cuda-cupti ={{ cuda_version }}
+        {% endif %}
     requirements:
       build:
         - cmake {{ cmake_version }}
         - ninja
       run:
+        {% if cuda_major == "11" %}
         - cudatoolkit {{ cuda_spec }}
+        {% else %}
+        - cuda-cudart {{ cuda_spec }}
+        {% endif %}
+        - librmm ={{ minor_version }}*=cuda{{ cuda_major }}*
         - libcudf ={{ minor_version }}
         - librmm ={{ minor_version }}
```

Comment:
```suggestion
        - libcudf ={{ minor_version }}*=cuda{{ cuda_major }}*
        - librmm ={{ minor_version }}*=cuda{{ cuda_major }}*
```

Reply:
Resolved this myself in 01b5c6b since there wasn't a diff nearby that I could use a suggestion to fix. :)